app/src/main/java/org/traccar/client/MainActivity.java

```diff
@@ -255,6 +277,10 @@ private void startTrackingService(boolean checkPermission, boolean permission) {
             startService(new Intent(this, TrackingService.class));
```

Comment:
Sure, because if user not enable providers, it will be useless. Traccar-client cannot update location and send it.